chore: ROOT-97: annotation reviews to SDK and docs

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/17624186083)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/566
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17478573496

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17478676825

jombooth committed:
Merge branch 'develop' into fb-ROOT-97/reviews-in-sdk

jombooth committed:
Merge branch 'develop' into fb-ROOT-97/reviews-in-sdk

jombooth committed:
empty commit

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17564368765

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-97/reviews-in-sdk'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17564368765

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17564614298

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17567607115

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-97/reviews-in-sdk'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17567607115

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17568038900

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17600689818)
> Successfully merged:  29 files changed, 528 insertions(+), 209 deletions(-)

jombooth committed:
Merge branch 'develop' into 'fb-ROOT-97/reviews-in-sdk'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17600689818

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17617756465

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-97/reviews-in-sdk'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17617756465

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17622277138)
> Successfully merged:  create mode 100644 docs/themes/v2/source/images/tags/vector.png

jombooth committed:
Merge branch 'develop' into 'fb-ROOT-97/reviews-in-sdk'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17622277138

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17624186083

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-97/reviews-in-sdk'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17624186083

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/17625441245
